app/src/router/index.ts

```
@@ -65,31 +64,40 @@ const routes: Array<RouteRecordRaw> = [
 
 const router = createRouter({ history: createWebHashHistory(), routes });
 
-// Handle login routing
+// Handle authentication and route guarding
 router.beforeEach(async (to, from, next) => {
   try {
-    const isAuthenticated = await ad4mConnect.isAuthenticated();
-    if (isAuthenticated) {
-      const appStore = useAppStore();
-      const { me } = storeToRefs(appStore);
+    const appStore = useAppStore();
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)